### PR TITLE
Bug 1885218: Add virtctl binary to kubevirt-plugin gating script

### PR DESCRIPTION
**Pull request by irosenzw**
Signed-off-by: Ido Rosenzwig <irosenzw@redhat.com>

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh [hunk 1]
```diff
@@ -47,3 +47,10 @@ metadata:
 data:
   feature-gates: "LiveMigration"
 EOF
+
+# Install virtctl binary and add in to PATH
+mkdir virtctl
+wget https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/virtctl-${KUBEVIRT_VERSION}-linux-x86_64 -O virtctl/virtctl
+chmod +x virtctl/virtctl
+
+export PATH="${PATH}:$(pwd)/virtctl"
```
Comment (suomiy) [inline frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh:56]: missing newline

Comment (irosenzw) [reply frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh]: fixed.

**Pull request merged**